compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ViewControllerBasedLifecycleOwner.uikit.kt

```diff
@@ -63,8 +70,8 @@ internal class ViewControllerBasedLifecycleOwner(
     private fun updateLifecycleState() {
         lifecycle.currentState = when {
             isDisposed -> State.DESTROYED
-            isViewAppeared && isAppForeground -> State.RESUMED
-            isViewAppeared -> State.STARTED
+            isViewAppeared && isAppForeground && isAppActive -> State.RESUMED
+            isViewAppeared && isAppForeground && !isAppActive -> State.STARTED
             else -> State.CREATED
```

Comment:
```suggestion
            !isViewAppeared || !isAppForeground -> State.CREATED
            !isAppActive -> State.STARTED
            isAppActive -> State.RESUMED
            else -> State.CREATED
```

Just a suggestion, needs to be rechecked

Reply:
I find Victor's variant more readable